src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :draftCourtOrderNumber="getFileNumber"
+        :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
+        @emitCourtNumber="setFileNumber($event)"
+        @emitPoa="setHasPlanOfArrangement($event)"
+        @emitValid="setValidCourtOrder($event)"
```

Comment:
These 3 event handlers need to be implemented in this component.

Reply:
Will do thanks!